examples/java/src/main/java/org/apache/beam/examples/WordCount.java

```diff
@@ -196,6 +197,26 @@ public String create(PipelineOptions options) {
       }
     }
 
+    /**
+     * Return default input file path according to runner type.
+     *
+     * <p><ul>
+     *   <li>SparkRunner:
+     *   .../src/test/resources/LICENSE</li>
+     *   <li>other runners:
+     *   gs://dataflow-samples/apache/LICENSE</li>
+     * </ul>
+     */
+    public static class InputFactory implements DefaultValueFactory<String> {
+      @Override
+      public String create(PipelineOptions options) {
+        if (options.getRunner().isAssignableFrom(SparkRunner.class)) {
```

Comment:
Yes, we want everyone use GCS by default, and FilnkRunner already support it. But SparkRunner can't use path starting with "gs://" as for as I know. This is one tmp solution in order to aggregate this E2E test to pre/post-submit test. Otherwise, SparkRunner side will be a blocker.